pycheribuild/projects/syzkaller.py

```diff
@@ -62,92 +69,135 @@ def setup_config_options(cls, **kwargs):
             "run-sysgen", show_help=True,
             help="Rerun syz-extract and syz-sysgen to rebuild generated Go "
                  "syscall descriptions.")
+        cls.use_go = cls.add_config_option("use-go", kind=GoType, default=GoType.DEFAULT,
+                                           enum_choice_strings=[t.value for t in GoType],
+                                           help="The Go type to run with.")
+
+    def get_path(self):
+        return (str(self.get_sdk_bindir()) + ":" +
+                str(self.config.dollar_path_with_other_tools))
+
+    def get_sdk_dir(self, config):
+        return config.cheri_sdk_dir
+
+    def get_sdk_bindir(self):
+        return self.config.cheri_sdk_bindir
 
     def __init__(self, config):
-        self._install_prefix = config.cheri_sdk_dir
-        self._install_dir = config.cheri_sdk_dir
+        self._install_prefix = self.get_sdk_dir(config)
+        self._install_dir = self.get_sdk_dir(config)
         self.destdir = ""
         super().__init__(config)
```

Comment:
There is no reason for late calling super `__init__()` I'd call it first so we have `self.config` available everywhere. Also we can call the getters everywhere instead of having private variables for those, but this is more flavor.